scripts/update_bibliography.sh

```diff
@@ -32,23 +32,41 @@ add_items_from_collection $root_collection
 echo "Got $(jq '. | length' <<< "$items") items"
 
 # Piece-wise processing for debugging:
-items=$(jq 'sort_by(.data.date)' <<< "$items")
-# echo "$items" > sorted.json
-items=$(jq 'map(.csljson)' <<< "$items")
-# echo "$items" > mapped.json
+# echo "$items" > 0-rawItems.json
+items=$(jq 'map(.csljson * .data)' <<< "$items")
```

Comment:
This seems to pull in a lot more `json` data per bibliography entry then the original version.  Do we need or want it?  We don't appear to be using the information.

Reply:
There were some entries that had fields in the .data object instead of the .csljson object. It seemed simplest to merge them at the creation of the bibliography.json file, since only the _actually_ referenced content ends up in the generated html. It also made some meta-analysis of the information easier, like entries with no url or duplicate ids.

Reply:
Also, the Portrait mode display issue is related to what I mentioned in today's meeting, about part of the table getting covered by the Hugo template's right-hand column.